vm/systemSmartContracts/eei.go

```diff
@@ -465,6 +465,14 @@ func (host *vmContext) CreateVMOutput() *vmcommon.VMOutput {
 		}
 		if outAcc.Nonce > 0 {
 			outAccs[addr].Nonce = outAcc.Nonce
+
+			if len(outAcc.CodeDeployerAddress) > 0 {
```

Comment:
not like this :). what you need to check if that blockchainHook.CurrentNonce() > 0 and set these.

Reply:
Done.